src/app.js

```diff
@@ -64,4 +64,10 @@ vorpal
     //Commands.shuffle(GameManager);
     callback();
   });
-
+  
```

Comment:
Add comments that describe what this command does

Reply:
Fixed and new commit